vizro-ai/src/vizro_ai/utils/helper.py

```diff
@@ -49,22 +60,29 @@ def _debug_helper(
 
 
 def _exec_code_and_retrieve_fig(
-    code: str, local_args: Optional[Dict] = None, is_notebook_env: bool = True
+    code: str, local_args: Optional[Dict] = None, show_fig: bool = False, is_notebook_env: bool = True
 ) -> go.Figure:
     """Execute code in notebook with correct namespace and return fig object.
 
     Args:
         code: code string to be executed
         local_args: additional local arguments
+        show_fig: boolean flag indicating if fig will be rendered automatically
         is_notebook_env: boolean flag indicating if code is run in Jupyter notebook
 
     Returns:
-        go.Figure
+        Plotly go figure
```

Comment:
```suggestion
        go.Figure
```
I think that is clearer actually seeing it's an element of `graph_objects`